gnome-image-installer/pages/install/gis-install-page.c

```diff
@@ -85,8 +85,8 @@ gis_install_page_prepare_read (GisPage *page, GError **error)
   basename = g_file_get_basename(priv->image);
   if (basename == NULL)
     {
-      *error = g_error_new(GIS_INSTALL_ERROR, 0, _("Image verification error: g_file_get_basename(\"%s\") failed"), priv->image);
-      return FALSE;
+      *error = g_error_new (GIS_INSTALL_ERROR, 0, _("Internal error"));
```

Comment:
These are only "original" as of 24 hours ago when I changed them from just "Image verification error." :smile: 

I'm in two minds on this: I don't want to add strings that are incomprehensible to translators; but you're right that it's easier to debug with more details. I guess I could use strings unmarked for translation for errors which should never occur?